src/accelerate/state.py

```diff
@@ -1112,6 +1109,14 @@ def deepspeed_plugin(self):
 
         return get_active_deepspeed_plugin(self)
 
+    def get_deepspeed_plugin(self, plugin_key: str):
```

Comment:
`plugin.plugin_key` is awkward - perhaps `plugin.name`?
```suggestion
    def get_deepspeed_plugin(self, name: str):
```
the fact that it's used as a dict key is irrelevant and doesn't have to be exposed in the api.